.github/workflows/ci.yml

```diff
@@ -146,3 +146,24 @@ jobs:
     - name: Create source and wheel dist
       run: |
         python -m build
+
+  server-package:
+    # This job checks if we can build our server package
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./server
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build the Docker image
+        run: |
+          docker buildx build -t basyx-python-sdk-http-server .
+      - name: Run container
+        run: |
+           docker run -d -p 8080:80 -v ./storage:/storage basyx-python-sdk-http-server
```

Comment:
The `-p` option is unnecessary, as we are not trying to access the container, we just want to see if it runs, same with the `-v` I think.